cgk_calcium_tools/filehandler.py

```diff
@@ -1717,3 +1866,30 @@ def parameters_for_isx(
             del copy_dict[key]
     copy_dict.update(to_update)
     return copy_dict
+
+
+def progress_bar(end_amount, description_step):
+    # Initialize progress bar
+    progress_bar = IntProgress(
+        value=0, 
+        min=0,
+        max=end_amount, 
+        style={
+            'bar_color': '#3385ff',
+            'description_width': 'auto'
+            },
+        description= f'{description_step} Movies: 0/{end_amount} movie(s)',
+        layout=Layout(width='45%')
+        )
+            
+    # Display the progress bar
+    display(progress_bar)
+    return progress_bar
+
+
+def update_progress_bar(progress_bar, description_step):
+    progress_bar.value += 1
+    if progress_bar.value == progress_bar.max:
+        progress_bar.description = f'{description_step} Movies: {progress_bar.value}/{progress_bar.max} Complete!'
+    else:
+        progress_bar.description = f'{description_step} Movies: {progress_bar.value}/{progress_bar.max} movie(s)'
```

Comment:
I would re implement the first function as an constructor class and the second as its method. In that way you don't have to pass two times `description_step` .

can you move it to another file? maybe `jupyter_outputs.py ` ?